d3a_api_client/setups/test_stocks_attribute_requirement.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from time import sleep
+
+from d3a_interface.utils import key_in_dict_and_not_none_and_greater_than_zero
+
+from d3a_api_client.redis_aggregator import RedisAggregator
+from d3a_api_client.redis_device import RedisDeviceClient
+from d3a_api_client.redis_market import RedisMarketClient
+
+
+class AutoAggregator(RedisAggregator):
```

Comment:
I think it would be better to change the name of this class. That's because we already have other classes with the same name, and since you're importing this in the `myco-api-client`, it will create confusion.